pkg/controller/namespace/namespace_controller.go

```diff
@@ -55,16 +57,36 @@ func NewNamespaceController(kubeClient client.Interface, resyncPeriod time.Durat
 		framework.ResourceEventHandlerFuncs{
 			AddFunc: func(obj interface{}) {
 				namespace := obj.(*api.Namespace)
-				err := syncNamespace(kubeClient, *namespace)
-				if err != nil {
-					glog.Error(err)
+				if err := syncNamespace(kubeClient, *namespace); err != nil {
+					if estimate, ok := err.(*contentRemainingError); ok {
+						go func() {
+							t := estimate.Estimate/2 + 1
```

Comment:
If the remaining content is going to take 10 secs to delete, then why not wait 10 secs? Why wait just 6 secs?